ivadomed/loader/utils.py

```diff
@@ -200,6 +200,16 @@ def get_subdatasets_subject_files_list(split_params, df, path_output, subject_se
         # Load subjects lists
         old_split = joblib.load(split_params["fname_split"])
         train_lst, valid_lst, test_lst = old_split['train'], old_split['valid'], old_split['test']
+
+        # convert into participant id into filename
+        df_train = df[df['participant_id'].isin(train_lst)]
```

Comment:
This behavior should only be for old `joblib` files with `participant_id`.
The new `joblib` files already contains the `filename` so this would fail.
So the function has to return lists of `filename` for both kind of inputs (joblib as participant_ids and joblib as filenames)